src/gui/tray/UserModel.cpp

```diff
@@ -45,6 +46,8 @@ User::User(AccountStatePtr &account, const bool &isCurrent, QObject *parent)
     connect(_account->account().data(), &Account::accountChangedDisplayName, this, &User::nameChanged);
 
     connect(FolderMan::instance(), &FolderMan::folderListChanged, this, &User::hasLocalFolderChanged);
+
+    connect(this, &User::guiLog, Logger::instance(), &Logger::guiLog);
```

Comment:
I'm not sure I understand how this is related to the rest of the patch.

Reply:
guiLog was being emmited but there was nothing connecting to ownCloudGui::slotShowTrayMessage
https://github.com/nextcloud/desktop/blob/b24e96a727e4fed83075f94e5e3592968fbe292b/src/gui/owncloudgui.cpp#L113

Reply:
I separated the changes in 2 commits, I guess makes it clear.

Reply:
It's likely me being daft on top of everything. ;-)